test/System/IntegrationTests/AssemblyConflictCheckerTest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.IO;
+using NUnit.Framework;
+using Dynamo.Applications;
+using System.Reflection;
+
+namespace IntegrationTests
+{
+    [TestFixture]
+    class AssemblyConflictCheckerTest
+    {
+        AssemblyConflictChecker ConflictChecker;
+        string testAssemblyConflictPath;
+
+        [Test]
+        public void ConflictOnAssemblyLoad()
+        {
+            testAssemblyConflictPath = Path.GetFullPath(Path.Combine(Directory.GetCurrentDirectory(), @"..\..\..\test\assemblyConflict"));
```

Comment:
not sure why git did not add the dlls under `test\assemblyConflict` to this PR ...
gitignore does not seem to exclude them...

Reply:
I think all .dlls are excluded, you may need to force add